app/main.py

```diff
@@ -1,3 +1,35 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os.path
+
```

Comment:
There is no need to place a blank line before the `decimal` import since it belongs to the same group